### DISCUSSION TITLE
交換圖工具

### DISCUSSION THREAD

**Discussion by dannypsnl** [FizzyElt/functional-programming#52] [Show and tell]
我之前有 [fork quiver](https://github.com/dannypsnl/quiver) 魔改成生成 forester 版本的 latex，可以用 `docker compose up` 跑起來

**Comment by dannypsnl** [comment]
使用的第三方工具是否要使用 wiki 紀錄？或是開卡片讓開發人員可以搜尋到即可？

**Reply by FizzyElt** [reply]
不太懂你的意思，是指這個工具的使用方式嘛？

**Reply by dannypsnl** [reply]
指記錄這些工具的存在

**Reply by FizzyElt** [reply]
放在 README 裡

**Comment by dannypsnl** [comment]
https://dannypsnl.me/k8s-0001.xml

紀錄一下我做了啥
